assets/src/edit-story/components/tabview/index.js

```diff
@@ -173,33 +173,31 @@ function TabView({
   getAriaControlsId,
   onTabChange = noop,
   tabs = [],
+  tab,
   label = '',
   shortcut = '',
-  initialTab,
   ...rest
 }) {
-  const [tab, setTab] = useState(initialTab || tabs[0]?.id);
```

Comment:
This was redundant state. The tabs (and current tab) are already being tracked in the library and inspector providers.